backend/core/renderers.py

```diff
@@ -37,3 +44,12 @@ class JSONRenderer(DRFJSONRenderer):
     """
 
     encoder_class = JSONEncoder
+
```

Comment:
what about the JSONEncoder class, can that be removed / does it need to be updated?